DuggaSys/tests/courseedservice-test.php

```diff
@@ -0,0 +1,373 @@
+/*
+----------------------------------------------------------
+    Courseedservice.php implement test file
+----------------------------------------------------------
+*/
+ 
+<?php
+ 
+include "../../Shared/test.php";
+ 
+$testsData = array(
+
+    /* Insert into course */
+
+    'Insert into course test' => array(
+        'expected-output' => '{
+            "LastCourseCreated": [],
+            "entries": [],
+            "versions": [],
+            "debug": "NONE!",
+            "writeaccess": null,
+            "motd": "UNK",
+            "readonly": 0
+          }',
+        'query-before-test-1' => "SELECT cid FROM course WHERE coursecode = 'G1337' ORDER BY coursecode DESC LIMIT 1",
+        'query-before-test-2' => "INSERT INTO course (coursecode,coursename,visibility) VALUES('G1337','Testing-Course',1)",
+        'query-before-test-3' => "DELETE FROM course WHERE coursecode = 'G1337' AND cid = ?",
+        'query-after-test-1' => "DELETE FROM course WHERE coursecode = 'G1337' AND coursename = 'Testing-Course'",
+        'query-after-test-2' => "DELETE FROM course WHERE coursecode = 'G1337' AND coursename = 'Testing-Course'",
+	'query-variables' => "blop",
+        'service' => 'https://cms.webug.se/root/G2/students/a21ponkr/LenaSYS/DuggaSys/courseedservice.php',
+        'service-data' => serialize(array(
```

Comment:
why is the login data supplied still not changed ? 
if you test the file with pretty print enabled, you can clearly see that the test fail.
![image](https://github.com/HGustavs/LenaSYS/assets/102600391/4ce61712-ed73-4c20-802c-aa06eb1c9e63)